css/app.css

```diff
@@ -47,10 +47,12 @@ aside h1 {
     width: 90%;
     background: #fff;
     padding: 25px 25px 0px 25px;
-    margin-top: -5rem;
     display: inline-block;
     z-index: 2;
 }
+div.blog-image-container + div.summary-blog-section {
+    margin-top: -5rem !important;
```

Comment:
Is the !important necessary? Usually a code smell

Reply:
It was originally as I had the selector as `div.blog-image-container + div`